src/build/build.ts

```diff
@@ -88,6 +88,15 @@ export async function build(
     logger.info(`(${buildName}) Building...`);
   });
 
+  if (options.addPushManifest) {
+    buildStream = buildStream.pipe(polymerProject.addPushManifest());
+    if (!options.addServiceWorker) {
+      logger.warn('Push manifest generated without a service worker. ' +
+      'HTTP2/Push relies on service workers to prevent duplicate file ' +
```

Comment:
I didn't either. I thought that the browser would cancel pushes if it already had the file in the cache.